### PR TITLE
ADDED PAYPAL BUTTONS

### PR DESCRIPTION
**Pull request by gregstula**
Co-authored-by: Gregory D. Stula <stulag@oregonstate.edu>

### CODE REVIEW

**Review by usenkok — CHANGES_REQUESTED** [review]
> Co-authored-by: Gregory D. Stula <stulag@oregonstate.edu> 

we should fix 

- signed in as guest

- title of site back to Alpacify

File: src/pages/index.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-            <Navbar.Brand href="/">Alpacify</Navbar.Brand>
+            <Navbar.Brand href="#home">Alpaca Sponsoring</Navbar.Brand>
         <Navbar.Toggle />
         <Navbar.Text>
-        <a href="/profile/">Alpaca Profile</a>
-        </Navbar.Text>  
+        <Link to="/profile/">View Our Alpacas</Link>
+        </Navbar.Text>
         <Navbar.Collapse className="justify-content-end">
+        <Navbar.Text>
```
Comment (usenkok) [inline src/pages/index.js:18]: @gregstula we accidentally added back in this "Signed in as Guest" :(

File: src/pages/index.js [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
 import alpaca from './alpaca-cute.jpeg'
 import "./styles.css"
 
-
 function AlpacaBar() {
     return (
         <div className="alpacaBar">
         <Navbar bg="dark" variant="dark">
-            <Navbar.Brand href="/">Alpacify</Navbar.Brand>
+            <Navbar.Brand href="#home">Alpaca Sponsoring</Navbar.Brand>
```
Comment (usenkok) [inline src/pages/index.js:12]: @gregstula I think this was an accident too

File: src/pages/profile.js [hunk 3]
```diff
@@ -55,7 +58,7 @@ export default () => {
           </div>  
           <div class="col-6">
             <ProfileText />
-            <Button variant="primary" size="lg">Sponsor now</Button>
```
Comment (usenkok) [inline src/pages/profile.js:58]: @SlinkEEE we sacrificed this button but think we should bring it back in a refactor ;)

File: src/pages/index.js (hunk 1 above)
Comment (usenkok) [reply src/pages/index.js:18]: here was Sheila's commit

> https://github.com/usenkok/CS361-W20-group33-agile-project/commit/7a3b98f46e4dfe8e0207441a5a4245fd0df8b2ae

File: src/pages/index.js (hunk 2 above)
Comment (usenkok) [reply src/pages/index.js:12]: https://github.com/usenkok/CS361-W20-group33-agile-project/commit/7a3b98f46e4dfe8e0207441a5a4245fd0df8b2ae

File: src/pages/profile.js (hunk 3 above)
Comment (gregstula) [reply src/pages/profile.js:58]: Ok we can add it back later <3

**Pull request merged**